_sass/_layout.scss

```diff
@@ -14,11 +14,32 @@ a {
   color: #39BBD5;
 }
 
-#main-blog p {
-  text-align: justify;
-}
-#main-blog h3 {
-  font-size: 1.5em;
-  text-transform: inherit;
-  font-weight: bold;
+#main-blog{
+    
+    h3 {
+        font-size: 1.5em;
+        text-transform: inherit;
```

Comment:
Why do we need this? Have you tried without it?

Reply:
I added when I wrote the first article and notice that h3 was smaller

Reply:
I mean the `text-transform` property. According to [Mozilla's documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform):

> The text-transform CSS property specifies how to capitalize an element's text. It can be used to make text appear in all-uppercase or all-lowercase, or with each word capitalized.

So I believe we don't need that here.

Reply:
We needet because in homepage is uppercase. I fixed in commit https://github.com/coopdevs/coopdevs.github.io/pull/12/commits/fe2e86669a5b4f8e85e741d62425bd2bc4fc87ed